.gitignore

```diff
@@ -5,10 +5,13 @@
 /phpunit.xml
 
 # Build data
-/build/
+src/build/
 
 # Composer PHAR
 /composer.phar
 
 # IDE prefs
 /.idea/
+
+.phpunit.result.cache
+composer.lock
```

Comment:
IMO the `composer.lock` should not be ignored by versioning system, it creates a state of the dependencies versions.